src/odemis/gui/model/__init__.py

```diff
@@ -1293,6 +1306,10 @@ def __init__(self, main):
 
         self.selected_scintillators = model.ListVA([])  # set of ints, overview images to be acquired
 
+        self.is_calib = model.BooleanVA(False)
```

Comment:
Maybe name that `is_calibrated` (assuming that's the meaning), to clarify better the difference with `is_calibrating`?

Reply:
I had another look at this. I actually had named it "calib" or "calib_1" etc. to be able to make the controllers generic by using the prefix attribute. If I make it a new name, I don't see atm how to generalize.
`        # get attribute that keeps track of calibration status
        setattr(self, "is_" + calib_prefix, getattr(tab_data, "is_" + calib_prefix))`